airflow/gcp/hooks/mlengine.py

```diff
@@ -275,3 +275,33 @@ def get_model(self, project_id: str, model_name: str) -> Optional[Dict]:
                 self.log.error('Model was not found: %s', e)
                 return None
             raise
+
+    def delete_model(self, project_id: str, model_name: str, delete_contents: bool = False) -> None:
+        """
+        Delete a Model. Blocks until finished.
```

Comment:
This documentation is not complete, but it imitates the style of the other methods. Documentation will update in a separate PR.

Reply:
Fine!